spec/models/form/lettings/questions/stock_owner_spec.rb

```diff
@@ -205,21 +208,26 @@
     end
 
     context "when user is support" do
-      let(:user) { create(:user, :support) }
+      user = nil
+      log = nil
+
+      before do
+        user = create(:user, :support)
+        log = create(:lettings_log)
+      end
```

Comment:
This is a pretty messy contrivance, to achieve the following:

- user and log need to be accessed in some tests cases
- user and log (and their related organisations) need to be instantiated in all test cases, even ones where they aren't referenced.

Open to better solutions to this problem!

Reply:
Oh hey, `let!` solves exactly this problem. Let's do that.